lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1157,6 +1157,14 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'If `true`, disables ActiveStorage instrumentation.'
         },
+        :disable_active_support => {
+          :default => false,
+          :public => true,
+          :type => Boolean,
+          :dynamic_name => true,
```

Comment:
I need to fix this in my PR as well, but I don't believe this new parameter name is dynamic, given that it exists in `newrelic.yml` and is statically known before agent startup.

Reply:
Fixed in: 278635542852a978fcb57364752500c445f237c6 

I also tackled a few others. Let me know if you'd like me to move those to a separate PR.